addon/components/variable-plugin/autofilled/edit.gts

```diff
@@ -0,0 +1,174 @@
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { NodeSelection, type SayController } from '@lblod/ember-rdfa-editor';
+import IntlService from 'ember-intl/services/intl';
+import AuFormRow from '@appuniversum/ember-appuniversum/components/au-form-row';
+import AuLabel from '@appuniversum/ember-appuniversum/components/au-label';
+import AuButton from '@appuniversum/ember-appuniversum/components/au-button';
+import AuCheckbox from '@appuniversum/ember-appuniversum/components/au-checkbox';
+import AuNativeInput from '@lblod/ember-rdfa-editor-lblod-plugins/components/au-native-input';
+import AuCard from '@appuniversum/ember-appuniversum/components/au-card';
+import AuHeading from '@appuniversum/ember-appuniversum/components/au-heading';
+import t from 'ember-intl/helpers/t';
+import { on } from '@ember/modifier';
+import { trackedReset } from 'tracked-toolbox';
+type Args = {
+  controller: SayController;
+};
+
+export default class AutoFilledVariableInsertComponent extends Component<Args> {
+  @service declare intl: IntlService;
+  @tracked label?: string;
+
+  // memo is the path to the trigger, which calls update
+  // see https://github.com/tracked-tools/tracked-toolbox?tab=readme-ov-file#trackedreset
+  // it's sort of badly explained in the docs, but it's a more generic version
+  // of localCopy, which we can't use here cause our derived state from the args
+  // is too complex to express as a string path
+  @trackedReset({
+    memo: 'args.controller.mainEditorState',
+    // using this as the first arg is special typescript syntax
+    // which allows you to explicitly type the this context, cause
+    // ts is not smart enough to know that
+    update(this: AutoFilledVariableInsertComponent) {
+      return this.convertToStringAttr;
+    },
+  })
+  convertToString: boolean = false;
+
+  @trackedReset({
+    memo: 'args.controller.mainEditorState',
+    update(this: AutoFilledVariableInsertComponent) {
+      return this.autofillKeyAttr;
+    },
+  })
+  autofillKey?: string;
+
+  get selectedVariable() {
+    const { selection } = this.controller.mainEditorState;
+    if (
+      selection instanceof NodeSelection &&
+      selection.node.type === this.controller.schema.nodes.autofilled_variable
+    ) {
+      return {
+        node: selection.node,
+        pos: selection.from,
+      };
+    } else {
+      return null;
+    }
+  }
+  get autofillKeyAttr() {
+    return this.selectedVariable?.node.attrs.autofillKey;
+  }
+  get convertToStringAttr() {
+    return this.selectedVariable?.node.attrs.convertToString;
+  }
+
+  get showCard() {
+    return !!this.selectedVariable;
+  }
+
+  get controller() {
+    return this.args.controller;
+  }
+
+  get schema() {
+    return this.args.controller.schema;
+  }
+
+  get documentLanguage() {
+    return this.controller.documentLanguage;
+  }
+
+  @action
+  updateLabel(event: InputEvent) {
```

Comment:
This isn't used, but I think we should have a way to edit the label too.

Reply:
Done